File: AkitaModel/FlagLevelLimits.cs
namespace Skyware.Lis.AkitaModel;

/// <summary>
/// Defines limit of flagging, e.g. up to alarm 1.
/// </summary>
public enum FlagLevelLimits : byte
{

    /// <summary>
    /// Do not use flags at all.
    /// </summary>
    NO_FLAG = 0,

    /// <summary>
    /// Up to Alarm 1 - High/Low.
    /// </summary>
    UP_TO_LOW_HIGH = 1,

    /// <summary>
    /// Up to Alarm 2 - Very High/Low.
    /// </summary>
    UP_TO_VERY = 2,

    /// <summary>
    /// Up to Alarm 3 - Ultra High/Low.
    /// </summary>
    UP_TO_ULTRA = 3

}

File: AkitaMotelTS/src/FlagLevelLimits.ts
/**
 * Skyware.Lis.AkitaModel
 */

/**
 * Defines limit of flagging, e.g. up to alarm 1.
 */
export enum FlagLevelLimits {
  /**
   * Do not use flags at all.
   */
  NO_FLAG = 0,

  /**
   * Up to Alarm 1 - High/Low.
   */
  UP_TO_LOW_HIGH = 1,

  /**
   * Up to Alarm 2 - Very High/Low.
   */
  UP_TO_VERY = 2,

  /**
   * Up to Alarm 3 - Ultra High/Low.
   */
  UP_TO_ULTRA = 3
}
